.github/workflows/validate-markdown.yaml

```diff
@@ -27,8 +26,8 @@ jobs:
       with:
         node-version: '20.x'
 
-    - name: Validate markdown
-      run: npx --yes mdv versions/3.*.md src/oas.md
+    - name: Lint work-in-progress spec
+      run: npx --yes markdownlint-cli2 --config spec.markdownlint.yaml src/oas.md
 
-    - name: Lint markdown 3.0.4, 3.1.1, and later
-      run: npx --yes markdownlint-cli --config .markdownlint.yaml versions/3.0.4.md versions/3.1.[^0].md versions/3.[2-9].*.md src/oas.md
+    - name: Lint other files
+      run: npx --yes markdownlint-cli2 --fix *.md
```

Comment:
I don't think we should `--fix` in CI, makes more sense to me to validate it here (as the job name implies). People can run `--fix` themselves if needed, and we'll review the fix.

Reply:
(but I'll edit this myself to get the rest of the changes into my original pull request)

Reply:
```suggestion
      run: npx --yes markdownlint-cli2 *.md
```